pandas/conftest.py

```diff
@@ -77,6 +78,24 @@ def observed(request):
     return request.param
 
 
+_all_arithmetic_operators = ['__add__', '__radd__',
+                             '__sub__', '__rsub__',
+                             '__mul__', '__rmul__',
+                             '__floordiv__', '__rfloordiv__',
+                             '__truediv__', '__rtruediv__',
+                             '__pow__', '__rpow__']
+if not PY3:
+    _all_arithmetic_operators.extend(['__div__', '__rdiv__'])
+
+
+@pytest.fixture(params=_all_arithmetic_operators)
+def all_arithmetic_operators(request):
+    """
+    Fixture for common arithmetic operations
```

Comment:
Maybe say that these are the dunder names? Otherwise unclear whether it's the like `__div__` or `div` or `operator.div`.

Reply:
fixed